### ISSUE TITLE
In FR1 we have the means for possible new volunteers to send an email to the SC to express an interest and this needs to be in FR2

### ISSUE THREAD

**Issue by edickens** [FreeUKGen/MyopicVicar#269]
Have we a watertight system for dealing with new volunteers?

**Comment by Captainkirkdawson** [comment]
Is this still valid

**Comment by edickens** [comment]
Yes.  And not only for volunteering.  There are questions on data
    availability and contacts from parishes asking to be involved. 
    Enhancement only.  They now use the link at the top of the FR1 PPP.
    EricD
    _____________________________________
Eric J Dickens
FreeREG Chairman

**Comment by Captainkirkdawson** [comment]
Sorry to say but I really do not understand what you want done.

There is a simple registration system that works. It could not be simpler.

**Comment by edickens** [comment]
Many ask the CC what is involved in transcribing before they start to 
make a commitment, which is what they will think they are doing if they 
start registration.

_____________________________________
Eric J Dickens
FreeREG Chairman

**Comment by PatReynolds** [comment]
Eric D's proposal that new transcribers should be allowed to indicate a preference, and the CC told that they are on their way, but the person be allocated to the Training Syndicate, assessed, and then transferred to county.

**Comment by edickens** [comment]
Hi Sean,
    Again what am I supposed to do?  I raised the suggestion, but it is
    a request from Coordinators for a way for a new volunteer to contact
    the SC prior to registering.  They usually ask if images are
    available for the parish in which they are interested, or they ask
    about what is involved and what is the commitment.  E
    _____________________________________
Eric J Dickens
FreeREG Chairman
    On 13/09/2015 12:03, sean-wood wrote:
    
    
      Assigned #269 to @edickens.
      —
        Reply to this email directly or view
          it on GitHub.

**Comment by sean-wood** [comment]
Hi Eric

I have re-assigned this to me to look into possible solutions and I will move this to needs review
Kind Regards


Sean Wood
Director and Principal Consultant
Sands View I.T. Consulting Ltd.

E: swood@sandsviewitconsulting.co.uk<mailto:swood@sandsviewitconsulting.co.uk>
T: +44 (0) 7771 824 315
W: www.sandsviewitconsulting.co.uk<http://www.sandsviewitconsulting.co.uk/>

**Comment by sean-wood** [comment]
Kirk / Eric

after reading this I have put together a document which I think describes what Eric is trying to get across.  I am trying to see if I can add it to the issue if not I will send seperately

**Comment by edickens** [comment]
I have been discussing this with Pat.  It involves the whole question of communication between researchers/transcribers and the CC, SC, developers and others.  I don't think programming is needed in FR.  We just need to monitor and refine the system.  E

**Comment by Captainkirkdawson** [comment]
Sean

I like the approach, very useful when there is disagreement.

This specific paper is missing parts of the background unfortunately.

When we designed FR (2 that was) we included a general registration page with 3 registration links. Transcriber. Technical and Researcher. The General page was linked to by a volunteering link on the footer. These still exist but are not linked to at this point. These were removed  a couple of months ago because people were registering prematurely. 

The point of registration entry is

http://freereg2.freereg.org.uk/userid_details/general

That described the three types of registration.

From there you go to a specific registration page. Each tailored to the type of registration. All had a link to the Transcription/Technical and Research agreements and acknowledgement was essential for registration. These agreements were removed at the request of the trustees
 
The first 2 are active links; the research one is not active but exists.

If you follow the link to the transcriber registration you will see the actual registration page.

This includes the list of syndicates that are looking for volunteers. The SC can change this status anytime

The transcriber gets an email and completes the registration by going to a link that is only active for 24 hrs. That link allows the setting of the password. 

A email goes directly to the SC when the transcriber registers and after completion of the registration.

In summary. The process was sign up and be looked after by your SC. This was more structured and automatic than FR1. This was especially important as we were expecting to register researchers and provide them with special services.

The concern was that SC started getting registrants before they and we were ready.

Then this story was started and you have seen some of the too and fro

**Comment by Captainkirkdawson** [comment]
PS The volunteer option in the Contact us was a stop gap while we were not using FR (2) for registration.
